components/uic_log/platform/posix/sl_log.cpp

```diff
@@ -42,8 +49,13 @@
 #include "config.h"
 #include "sl_status.h"
 
+#ifdef BOOST_LOG_USE_NATIVE_SYSLOG
+static char process_name[48] = {0};
```

Comment:
why 48 ? I would use a constant like MAX_PATH

Reply:
This is limit in syslog itself.
The syslog-ng OSE application will truncate the following fields:
If APP-NAME is longer than 48 characters it will be truncated to 48 characters.
https://www.syslog-ng.com/technical-documents/doc/syslog-ng-open-source-edition/3.18/administration-guide/8

Reply:
ok makes sense if this is defined in a constant somewhere may you use it or add reference link